app/controllers/coronavirus_form/have_you_been_made_unemployed_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::HaveYouBeenMadeUnemployedController < ApplicationController
+  skip_before_action :check_first_question
```

Comment:
Just realised that you'll only need this for the first question

Reply:
Thanks! I'll remove it.